homeassistant/components/linkplay/const.py

```diff
@@ -0,0 +1,77 @@
+"""LinkPlay constants."""
+
+from linkplay.consts import EqualizerMode, LoopMode, PlayingMode, PlayingStatus
+
+from homeassistant.components.media_player import (
+    MediaPlayerEntityFeature,
+    MediaPlayerState,
+    RepeatMode,
+)
+from homeassistant.const import Platform
+
+DOMAIN = "linkplay"
+CONTROLLER = "controller"
+BRIDGE_DISCOVERED = "bridge_discovered"
```

Comment:
Dispatcher signals are global. 
```suggestion
BRIDGE_DISCOVERED = "linkplay_bridge_discovered"
```